calisphere/views.py

```diff
@@ -933,6 +933,51 @@ def collectionView(request, collection_id):
     return render(request, 'calisphere/collectionView.html', context)
 
 
+def collectionTitleFacet(request, collection_id):
+    collection_url = 'https://registry.cdlib.org/api/v1/collection/' + collection_id + '/'
+    collection_details = json_loads_url(collection_url + '?format=json')
+    if not collection_details:
+        raise Http404("{0} does not exist".format(collection_id))
+
+    for repository in collection_details.get('repository'):
+        repository['resource_id'] = repository.get('resource_uri').split(
+            '/')[-2]
+
+    params = request.GET.copy()
+    context = searchDefaults(params)
+    # facet=true&facet.query=*&rows=0&facet.field=title_ss&facet.pivot=title_ss,collection_data"
+    solrParams = {
+        'facet': 'true',
+        'rows': 0,
+        'facet_field': 'title_ss',
+        'fq': 'collection_url:"{}"'.format(collection_url),
+        'facet_limit': '-1',
+        'facet_mincount': 1,
+        'facet_sort': 'count',
+    }
+    solr_search = SOLR_select(**solrParams)
+
+    titles = solr_search.facet_counts.get('facet_fields').get('title_ss')
+    unique = len(titles)
+    records = sum(titles.values())
+    ratio = unique / records
+    context.update({'titles': titles, 'unique': unique, 'records': records, 'ratio': ratio})
+
+    context.update({
+        #'meta_robots': "noindex,follow",
+        'title': 'title reports should have unique titles too',
+        'meta_robots': False,
```

Comment:
I think this should be 'None', per https://moz.com/learn/seo/robots-meta-directives, but also I'm not sure how a bot would find these pages since they're not linked anywhere.

```suggestion
        'meta_robots': 'None',
```